stacks-signer/src/signerdb.rs

```diff
@@ -144,6 +144,17 @@ impl SignerDb {
         )
     }
 
+     /// Get the sqlite version from the database
+     pub fn get_sqlite_version(
```

Comment:
I've added this method to display the sqlite version used. I'm assuming these DBs should have the same version.